lib/LinkEmbedder/Link.pm

```diff
@@ -382,9 +387,9 @@ __DATA__
 <a class="le-<%= $l->type %> le-provider-<%= lc $l->provider_name %>" href="<%= $l->url %>"><%= Mojo::Util::url_unescape($l->url) %></a>
 % }
 @@ video.html.ep
-<video class="le-<%= $l->type %> le-provider-<%= lc $l->provider_name %>" height="640" width="480" preload="metadata" controls>
-% for my $s (@{$l->{sources} || []}) {
-  <source src="<%= $s->{url} %>" type="<%= $s->{type} || '' %>">
-% }
-  <p>Your browser does not support the video tag.</p>
-</video>
+<div class="le-<%= $l->type %> le-provider-<%= lc $l->provider_name %>">
+  <video preload="metadata" controls>
+    <source src="<%= $l->url %>" type="<%= $l->mimetype || '' %>">
+    <p>Your browser does not support the video tag.</p>
```

Comment:
This is not really related to your PR, but I was wondering if this message should be changed into something like this:

```
<p>Your browser is unable to play the <%= $l->mimetype || 'video' %> element.</p>
```

Reply:
Good idea.